src/data/redirects.json

```diff
@@ -1,7 +1,7 @@
 [
   {
-    "from": "/fellows",
```

Comment:
why don't we need this redirect anymore?

Reply:
Thank you for catching this! I thought I copied and pasted the json object before changing it to the covid redirect, but it looks like i just changed an existing line. I will fix this right away!